sunpy/util/util.py

```diff
@@ -92,12 +92,12 @@ def expand_list(inp):
     ----------
     * https://stackoverflow.com/questions/2185822/expanding-elements-in-a-list/2185971#2185971
     """
-    return [item for item in expand_list_generator(inp)]
+    return list(expand_list_generator(inp))
 
 
 def expand_list_generator(inp):
     for item in inp:
-        if isinstance(item, tuple | list | UserList):
+        if isinstance(item, (list | tuple | Iterator)):
```

Comment:
This syntax is not supported for earlier versions of python, should I refactor it?

Reply:
We don't want to flattened strings or byte string, that's why we have used `not isinstance(item, (bytes, str))` so that it goes through `yield item` and not through `yield from expand_list_generator(item)`

Reply:
> This syntax is not supported for earlier versions of python, should I refactor it?

sunpy main supports python 3.10 or above, I don't think this should be a problem.

> We don't want to flattened strings or byte string, that's why we have used `not isinstance(item, (bytes, str))` so that it goes through `yield item` and not through `yield from expand_list_generator(item)`

Oh I didn't add a test with a longer string, I should add that to check.